.github/workflows/push.yml

```diff
@@ -80,6 +80,10 @@ jobs:
       - name: Run tests
         run: make test
 
+      - name: Verify -inprocess feature of acceptance test by running one test
+        # QQQ this will succeed if test was not found, unfortunately
```

Comment:
What does it test if it doesn't call into the CLI?

Reply:
Removed this; added explicit test case instead.